deploy/base/model.yaml

```diff
@@ -1,2 +1,4 @@
 small: embeddings_small_2023_05_24_23_17_03.csv
 medium: embeddings_medium_2023_05_24_22_15_24.csv
+finetuned_alternatives_model: ft_alt_all-MiniLM-L6-v2_v20230623171244
+finetuned_final_model: ft_final_ft_alt_all-MiniLM-L6-v2_v20230623171244_v20230623171328
```

Comment:
I slightly wonder if we should do `ft_final_v20230623171328.ft_alt_all-MiniLM-L6-v2_v20230623171244.csv`

so the center part is the full file-name of the base.

Reply:
Sure. Adopted this convention with a slight modification: `ft_final_v20230623171328.ft_alt_v2_v20230623171244.all-MiniLM-L6`. For consistency, the chain shows the base model (all-MiniLM-L6) at the end as well.